src/main/java/seedu/address/logic/parser/modulelistparsers/EditModuleParser.java

```diff
@@ -95,7 +75,10 @@ public EditModuleCommand parse(String args) throws ParseException {
         if (tags.isEmpty()) {
             return Optional.empty();
         }
-        Collection<String> tagSet = tags.size() == 1 && tags.contains("") ? Collections.emptySet() : tags;
```

Comment:
Request feedback :
Not sure of the purpose of this if else statement, but it causes editmodule to accept the first tag even if its empty e.g
editmodule t/. I have modified it such that having this condition will throw an exception,  if there are no regression bugs then it should be fine. Main issue is whether this if else statement serves any other purpose.

Reply:
If I am not wrong the intended purpose of the command `editmodule t/` is to allow users to delete all their tags? The rationale is that since tag is an optional field, users should be able to delete them.

Reply:
Okay noted. If that is the case then we should note it down in the userguide as well.

Reply:
Yes, it is in the user guide already.